pydantic/decorators.py

```diff
@@ -298,6 +295,8 @@ def root_validator(
     Decorate methods on a model indicating that they should be used to validate (and perhaps modify) data either
     before or after standard model parsing/validation is performed.
     """
+    if allow_reuse is True:  # pragma: no cover
+        warn('`allow_reuse` is deprecated and will be ignored', DeprecationWarning)
```

Comment:
```suggestion
        warn('`allow_reuse` is deprecated and will be ignored; it should no longer be necessary', DeprecationWarning)
```
Can phrase this differently, but I think a good idea to tell people that their code will probably "just work" if they drop the `allow_reuse`, clarifying that it's not deprecated and will break their code.

Reply:
(And same note above)